Jumpscale/data/nacl/NACL.py

```diff
@@ -265,24 +255,26 @@ def _keys_generate(self, words=None):
         will load in this class
         """
         if words:
-            key2 = j.data.encryption.mnemonic.to_entropy(words)
+            # generate key from mnemonic words
+            entropy = j.data.encryption.mnemonic.to_entropy(words)
+            key = nacl.signing.SigningKey(entropy).encode()
```

Comment:
what is the reason we now have to go via the `nacl` type and encode, while previously we hadn't?

Reply:
This is just the exact opposite of the word property. Now you actually generate the same signing key when providing the words.

Reply:
Ok will re-read again.